config/ext_jenkins-release_privatek8s.yaml

```diff
@@ -0,0 +1,692 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+# serviceAccountAgent:
+#   create: true
+#   name: jenkins-agent
+serviceAccountAgent:
+  create: false
+rbac:
+  create: true
+  readSecrets: true
+persistence:
+  enabled: true
+  size: 50Gi
+  storageClass: managed-csi-premium-retain
+agent:
+  componentName: "agent"
+networkPolicy:
+  # As of today, 2020-06-19, network policy is not supported for windows node
+  # https://docs.microsoft.com/en-us/azure/aks/windows-node-limitations#are-all-features-supported-with-windows-nodes
+  # This is a blocker for the release environment as in the current state,
+  # windows containers are not allowed to connect on main Jenkins.
+  # I disable it for now
+  # enabled: true
+  enabled: false
+  internalAgents:
+    allowed: true
+    namespaceLabels:
+      name: "jenkins-release"
+controller:
+  image: jenkinsciinfra/jenkins-lts
+  tag: 0.28.0-2.387.1
+  imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: linux
+    kubernetes.azure.com/agentpool: linuxpool
+  resources:
+    limits:
+      cpu: "2"
+      memory: "4Gi"
+    requests:
+      cpu: "2"
+      memory: "4Gi"
+  probes:
+    startupProbe:
+      initialDelaySeconds: 120
+    livenessProbe:
+      initialDelaySeconds: 120
+    readinessProbe:
+      initialDelaySeconds: 120
+  testEnabled: false
+  numExecutors: 0
```

Comment:
[async] To be reported on infra & weekly?

Reply:
This line is not needed: 

- Current configuration sets `controller.JCasc.defaultConfig` to `false` so the `controller.numExecutors` is never used (https://github.com/jenkinsci/helm-charts/blob/12a3c0992b9be68490e8944211506a22c5814b5f/charts/jenkins/templates/jcasc-config.yaml#L25-L45 and https://github.com/jenkinsci/helm-charts/blob/aff05875d84a7d79f010abc8756a11940804131b/charts/jenkins/templates/config.yaml#L60-L62 , the if is false so the template for the jcasc "defaults" which [consumes `numExecutors`](https://github.com/jenkinsci/helm-charts/blob/d0aa61f5921fb937e6f0a75ee3c1cf1f86cd2b18/charts/jenkins/templates/_helpers.tpl#L104-L121) is never called)

Reply:
besides, if we keep the config in the JCasc configuration, it will always override

Reply:
Yup, it didn't do anything on a new apply, the (empty) release controller had 2 executors, reverted this move.

Reply:
> Current configuration sets controller.JCasc.defaultConfig to false so the controller.numExecutors is never used ([jenkinsci/helm-charts@12a3c09/charts/jenkins/templates/jcasc-config.yaml#L25-L45](https://github.com/jenkinsci/helm-charts/blob/12a3c0992b9be68490e8944211506a22c5814b5f/charts/jenkins/templates/jcasc-config.yaml#L25-L45) and [jenkinsci/helm-charts@aff0587/charts/jenkins/templates/config.yaml#L60-L62](https://github.com/jenkinsci/helm-charts/blob/aff05875d84a7d79f010abc8756a11940804131b/charts/jenkins/templates/config.yaml#L60-L62) , the if is false so the template for the jcasc "defaults" which [consumes numExecutors](https://github.com/jenkinsci/helm-charts/blob/d0aa61f5921fb937e6f0a75ee3c1cf1f86cd2b18/charts/jenkins/templates/_helpers.tpl#L104-L121) is never called)

I'll probably make a PR on the official chart to add a notice about that, not obvious when reading https://github.com/jenkinsci/helm-charts/blob/12a3c0992b9be68490e8944211506a22c5814b5f/charts/jenkins/VALUES_SUMMARY.md